packages/cli/test/copy-assets/assembleAarBundle.test.ts

```diff
@@ -168,7 +168,7 @@ describe("assembleAarBundle", () => {
           /[/\\]node_modules[/\\].rnx-gradle-build[/\\]rnx-kit_react-native-auth[/\\]settings.gradle$/
         ),
         expect.stringMatching(
-          /include\(":rnx-kit_react-native-auth"\)\nproject\(":rnx-kit_react-native-auth"\).projectDir = file\(".*?[/\\]node_modules[/\\]@rnx-kit[/\\]react-native-auth[/\\]android"\)/
+          /include\(":rnx-kit_react-native-auth"\)\nproject\(":rnx-kit_react-native-auth"\).projectDir = file\(".*?[/\\\\]node_modules[/\\\\]@rnx-kit[/\\\\]react-native-auth[/\\\\]android"\)/
```

Comment:
To allow multiple backslashes, we want to add `+`: 
```suggestion
          /include\(":rnx-kit_react-native-auth"\)\nproject\(":rnx-kit_react-native-auth"\).projectDir = file\(".*?[/\\]+node_modules[/\\]+@rnx-kit[/\\]+react-native-auth[/\\]+android"\)/
```

Reply:
Alternatively, I think `{1,2}` should also work.

Reply:
Oh just saw this. Sorry I haven't wrote RegEx for a while. Kind of forgetting things. I actually want to disallow `\` which is invalid, and allow `\\` only. I'm hoping my next try works :)

Reply:
@tido64 I think it's ready now. Thanks for your quick response and help on this one!